### PR TITLE
[eclipse-temurin]  update jdk8

### PR DESCRIPTION
**Pull request by zdtsw**
update from jdk8u345 to jdk8u352

### CODE REVIEW

**Comment by zdtsw** [comment]
I will remove ubi as well

File: library/eclipse-temurin [hunk 1]
```diff
@@ -6,116 +6,128 @@ GitRepo: https://github.com/adoptium/containers.git
...
 File: Dockerfile.releases.full
 
-Tags: 8u345-b01-jdk-windowsservercore-ltsc2022, 8-jdk-windowsservercore-ltsc2022, 8-windowsservercore-ltsc2022
-SharedTags: 8u345-b01-jdk-windowsservercore, 8-jdk-windowsservercore, 8-windowsservercore, 8u345-b01-jdk, 8-jdk, 8
+Tags: 8u352-b08-jdk-ubi9-minimal, 8-jdk-ubi9-minimal, 8-ubi9-minimal
+Architectures: amd64, arm64v8, ppc64le
+GitCommit: fa65b46d19f4db0e1c8736c3d7b2a8392dd58b1b
+Directory: 8/jdk/ubi/ubi9-minimal
+File: Dockerfile.releases.full
+
```
Comment (zdtsw) [inline library/eclipse-temurin:39]:
```suggestion
```

File: library/eclipse-temurin [hunk 2]
```diff
@@ -6,116 +6,128 @@ GitRepo: https://github.com/adoptium/containers.git
...
 File: Dockerfile.releases.full
 
-Tags: 8u345-b01-jre-windowsservercore-ltsc2022, 8-jre-windowsservercore-ltsc2022
-SharedTags: 8u345-b01-jre-windowsservercore, 8-jre-windowsservercore, 8u345-b01-jre, 8-jre
+Tags: 8u352-b08-jre-ubi9-minimal, 8-jre-ubi9-minimal
+Architectures: amd64, arm64v8, ppc64le
+GitCommit: fa65b46d19f4db0e1c8736c3d7b2a8392dd58b1b
+Directory: 8/jre/ubi/ubi9-minimal
+File: Dockerfile.releases.full
+
```
Comment (zdtsw) [inline library/eclipse-temurin:102]:
```suggestion
```

**Comment by gdams** [comment]
LGTM

**Pull request merged**